docs/docs/intro.md

```diff
@@ -13,31 +13,34 @@ feature.
 
 ## Features
 
-At this point, ZMK is still missing many features. Currently, the working bits
-include:
-
-- Wireless connectivity via BLE HID Over GATT (HOG)
-- USB connectivity
-- Low active power usage
-- Split keyboard support
-- [Keymaps and layers](behavior/layers)
-- [Hold-tap](behavior/hold-tap) (which includes [mod-tap](behavior/mod-tap), [layer-tap](behavior/layers))
-- [Basic HID over USB](behavior/key-press)
-- [Basic consumer (media) keycodes](behavior/key-press#consumer-key-press)
-- [Encoders](feature/encoders)
-- Basic [OLED display support](feature/displays)
-- [RGB Underglow](feature/underglow)
-
-## Missing Features
-
-- One Shot Keys
-- Combo keys
-- Macros
-- Complete split support (encoders and RGB are not supported on the 'peripheral' side)
-- Battery reporting
-- Low power sleep states
-- Low power mode (to toggle LEDs and screen off)
-- Shell over BLE
+At this point, ZMK is still missing many features compared to more developed firmware such as QMK. Here is a table comparing supported ZMK and QMK features:
```

Comment:
Perhaps instead of "more developed", we could say "advanced" or even just "popular"?